### ISSUE TITLE
[Array firstObject] should be converted to .first

### ISSUE THREAD

**Issue by drkameleon** [drkameleon/iswift#76]
See: https://github.com/drkameleon/iswift/issues/75#issue-109621084

**Comment by drkameleon** [comment]
Hmm...

How can we differentiate between an `NSArray` (which sure has a `firstObject` property) and a Swift array (which has a `first` property)?

https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSArray_Class/#//apple_ref/occ/instp/NSArray/firstObject

Given that conversion is from Objective-C to Swift, couldn't we consider that all "initial" arrays would be treated as `NSArray`s?

![screen shot 2015-10-03 at 2 42 09 pm](https://cloud.githubusercontent.com/assets/1265028/10262942/417d840c-69dd-11e5-8113-b20912e3392a.png)

**Comment by charlieMonroe** [comment]
NSArray is already converted to Swift's Array in iSwift, isn't it. So it shouldn't be an issue. E.g.:

```
NSArray *array = @[ @"2", @"3" ];
NSString *first = [array firstObject];
```

will become (after running through iSwift):

```
var array: Array = ["2","3"]
var first: String = array.firstObject()
```

You shouldn't be interacting with NSArray from Swift, unless absolutely necessary...

**Comment by charlieMonroe** [comment]
Oh, and the same goes with `[array lastObject]` vs. `array.last`.

**Comment by drkameleon** [comment]
Yep, indeed. It's going in my to-do list.

----

P.S. I'm currently working on the next release (probably coming on Monday), so... every possible issue report, idea, anything is super-welcome! :)

**Comment by drkameleon** [comment]
Fixed as of the upcoming 1.9.

----

**Example:**

```objective-c
@implementation aClass

- (void)someFunc {
    NSArray* a = @[@"one", @"two", @"three"];
    NSString* first = [a firstObject];
    NSString* last = [a lastObject];
}

@end
```

**Output:**

```swift
class aClass {

	func someFunc() {
		var a: Array = ["one","two","three"]
		var first: String = a.first
		var last: String = a.last
	}

}
```
